README.md

```diff
@@ -1,40 +1,78 @@
-The ATAT Web API repo contains a set of services which enable ATAT Provisioning into vendor environments by way of the [ATAT CSP Specification](https://github.com/dod-ccpo/atat-csp-orchestration/blob/main/provisioning/atat_provisioning.yaml). It is implemented in a stateless fashion, using SQS Queues to pass information back to integrators. Initially, this API is only meant to be accessed by the Service Now component of ATAT, but it may be opened up to other integrators in the future.
+# ATAT Web API
 
-## Useful commands
+This repository contains two components that together act as a bridge between the internal front-end of the ATAT
+application and vendor implementations of the
+[ATAT CSP Orchestration API](https://github.com/dod-ccpo/atat-csp-orchestration). Any such implementors should
+consider this repository to contain purely "private" implementation details; however, it does serve as a sample
+client of the ATAT CSP Orchestration API and may be interesting to review for that purpose.
 
- * `npm run build`   compile typescript to js
- * `npm run watch`   watch for changes and compile
- * `npm run test`    perform the jest unit tests
- * `cdk deploy`      deploy this stack to your default AWS account/region
- * `cdk diff`        compare deployed stack with current state
- * `cdk synth`       emits the synthesized CloudFormation template
+Generally, the application is implemented to operate in a stateless fashion (with some transient state) leveraging
+SQS Queues, Step Functions State Machines, and a REST API. The private, internal-facing API is referred to as HOTH
+and serves a helpful API for other internal implementation details of the ATAT client application. Currently, HOTH
```

Comment:
Shall we define the acronym here? :)

Reply:
Done! Had to reflow the text a bit so the commit isn't quite as clean as just adding
the word.